selenium-pages/src/test/java/com/epam/engx/selenium/pages/google/GoogleCloudPageTest.java

```diff
@@ -2,10 +2,13 @@
 
 import com.epam.engx.selenium.pages.SmokeTest;
 import com.epam.engx.selenium.pages.browser.Browser;
+import com.epam.reportportal.junit5.ReportPortalExtension;
 import org.junit.jupiter.api.*;
+import org.junit.jupiter.api.extension.ExtendWith;
 
 import static org.assertj.core.api.BDDAssertions.then;
 
+@ExtendWith(ReportPortalExtension.class)
```

Comment:
FIY
Using for instance TestNG you can enable ReportPortal for the whole test suite via an appropriate listener.

Reply:
I hope to explore the possibilities of the report portal in more detail soon.